src/containers/AsideNavigation/AsideNavigation.tsx

```diff
@@ -15,18 +18,20 @@ import userChecked from '../../assets/icons/user-check.svg';
 import settingsIcon from '../../assets/icons/settings.svg';
 import supportIcon from '../../assets/icons/support.svg';
 
-import {UserSettings} from '../UserSettings/UserSettings';
-
-import routes, {createHref} from '../../routes';
-
 import {logout} from '../../store/reducers/authentication';
 import {getParsedSettingValue, setSettingValue} from '../../store/reducers/settings/settings';
+import {TENANT_GENERAL_TABS_IDS} from '../../store/reducers/tenant/constants';
```

Comment:
Its no longer tabs. Maybe it should be renamed in separate PR or here in additional commit?